doc/release_notes/source/release_note.rst

```diff
@@ -163,6 +165,9 @@ List of Supported Peripheral Devices and Features
 - **Ethos U55**:
   Pairs with Cortex-M55 for AI/ML acceleration using Arm v8.1 and Helium MVE.
 
+- **Ethos U85**:
```

Comment:
Now this list is getting confusing as you don't make a distinction between E7(, E5,E3) and E8(,E6, E4),  and B1.